package.json

```diff
@@ -5,6 +5,7 @@
   "workspaces": [
     "content-types/*",
     "packages/*",
```

Comment:
Is packages/* needed anymore?

Reply:
good callout, yep! i'm going to migrate some packages from other repos into this one and use the `packages` workspace for them.